### ISSUE TITLE
Blockquotes not fully formatted in PDF form

### ISSUE THREAD

**Issue by rlabbe** [ipython/nbconvert#203]
If you start a paragraph with '>' it becomes blockquoted

> like this

In the notebooks it is displayed indented with a bar to the left, the same as is done above. When using nbconvert to convert to PDF the bar is not displayed in the PDF output. This leaves it quite unclear that the content is a sidebar and not part of the text,especially if it is short (and thus looks like a normal indented paragraph). When exporting to HTML the bar is displayed in the HTML output.

**Comment by jakobgager** [comment]
Have you tried adding an empty line before the block quote? pandoc seems to require an empty line before such blocks to correctly parse them. The markdown parser of the notebook does not have this limitation.
Btw. this repo is deprecated. You should post issues either to ipython/ipython or better jupyter/jupyter_nbconvert.

**Comment by rlabbe** [comment]
Yes, this happens with empty lines before (and after) the block quote

**Comment by jakobgager** [comment]
I think I misunderstood your question. Actually quoting works as expected (indent on both sides of the text) but you want a side bar when quoting in pdf just like it is present with html? 
This is not default, and not what quotes look like in LaTeX. I fear to get such additional items, you will have to build a (quite tricky) custom template with a custom filter or alter the resulting tex file. For some ideas to the former have a look at [this SO question](http://stackoverflow.com/q/4018493/2870069).
